java/anaplan-connect-library/ac-impl/ac-impl-default/src/main/java/com/anaplan/client/DefaultServiceProvider.java

```diff
@@ -28,5 +29,17 @@ public static Service getService(ConnectionProperties properties)
     return new Service(properties, authenticator, apiProvider);
   }
 
+  /**
+   * Get the authenticator based on properties
+   * @param properties the connection properties
+   * @return {@link DeviceAuthenticator}
+   */
+  public static DeviceAuthenticator getDeviceAuthenticator(ConnectionProperties properties) {
```

Comment:
Don't see this being used anywhere please check.

Reply:
It was not used anymore, method has been removed